compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Error.hs

```diff
@@ -7,11 +7,14 @@ module DA.Daml.LF.TypeChecker.Error(
     TemplatePart(..),
     UnserializabilityReason(..),
     SerializabilityRequirement(..),
-    errorLocation
+    errorLocation,
+    toDiagnostic
```

Comment:
Haskell allows trailing commas here. I like that since it helps fighting future diff noise. 😃

Reply:
Fair, I tend to find it easier to never use them in Haskell than having to remember the one place where they are allowed but I’ve changed it now :slightly_smiling_face: